src/components/editor/property/InputLang.jsx

```diff
@@ -4,7 +4,7 @@ import React, { Component } from 'react'
 import { Typeahead } from 'react-bootstrap-typeahead'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
-import { setLang } from '../../actions/index'
+import { setLang } from '../../../actions/index'
```

Comment:
Could the path here not be `'components/actions/index'`? I assumed that's what the changes to `package.json` and `.eslintrc.js` would enable. No?

Reply:
Or is it more that you're not sanguine about committing to nuking each and every `'../'` import in the codebase? That'd be reasonable. Maybe we could get eslint to auto-fix that for us?

Reply:
Yes, I was trying only to change the `__test__` imports in this PR, but I think we could change the imports in `src` too.